themes/default/content/registry/packages/harbor/installation-configuration.md

```diff
@@ -10,7 +10,6 @@ The Pulumi `Harbor` provider is available as a package in all Pulumi languages:
 
 * JavaScript/TypeScript: [`@pulumiverse/harbor`](https://www.npmjs.com/package/@pulumiverse/harbor)
 * Python: [`pulumiverse-harbor`](https://pypi.org/project/pulumiverse-harbor/)
-* Go: [`github.com/pulumiverse/pulumi-harbor/sdk/v3`](https://pkg.go.dev/github.com/pulumiverse/pulumi-harbor/sdk/v3)
```

Comment:
isn't this at https://github.com/pulumiverse/pulumi-harbor/tree/main/sdk

Reply:
@ringods are you good with linking direct to github instead of to the go package at pkg.go?

Reply:
It's the provider I manage, and yes good to go for GitHub!